front/style/_committee.scss

```diff
@@ -330,6 +330,17 @@
                 }
             }
         }
+
+        &-description {
```

Comment:
I think the lists look a bit tight within the event description paragraph. Do you mind adding breathing room above and below the UL/OLs?